client/src/components/DataFiles/DataFilesModals/DataFilesUploadModal.js

```diff
@@ -3,17 +3,26 @@ import { useSelector, useDispatch } from 'react-redux';
 import { useHistory, useLocation } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import { v4 as uuidv4 } from 'uuid';
-import {
-  Button,
-  Modal,
-  ModalHeader,
-  ModalBody,
-  ModalFooter,
-  Table
-} from 'reactstrap';
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
 import { LoadingSpinner, FileInputDropZone } from '_common';
 import { findSystemOrProjectDisplayName } from 'utils/systems';
-import { FileLengthCell } from '../DataFilesListing/DataFilesListingCells';
+import DataFilesUploadModalListingTable from './DataFilesUploadModalListing/DataFilesUploadModalListingTable';
+
+import './DataFilesUploadModal.module.scss';
+
+export const DIRECTION_CLASS_MAP = {
+  vertical: 'is-vert',
+  horizontal: 'is-horz'
+};
+export const DEFAULT_DIRECTION = 'vertical';
+export const DIRECTIONS = ['', ...Object.keys(DIRECTION_CLASS_MAP)];
+
+export const DENSITY_CLASS_MAP = {
+  compact: 'is-narrow',
+  default: 'is-wide'
+};
+export const DEFAULT_DENSITY = 'default';
+export const DENSITIES = ['', ...Object.keys(DENSITY_CLASS_MAP)];
 
 const DataFilesUploadStatus = ({ i, removeCallback }) => {
```

Comment:
This code now appears unused after moving into `DataFilesUploadModalListingTable`

Reply:
This code is needed here because it affects Modal layout.

Reply:
Could you point as to where? `DataFilesUploadStatus` doesn't appear referenced in this file

Reply:
You are right, I thought you were talking about those constants above that function.